Makefile

```diff
@@ -156,13 +156,14 @@ else # COORD
 		override COORD:=oss
 	endif
 
-	ifeq ($(COORD),oss) # OSS Coordinator
+	ifeq ($(COORD),oss) # OSS (community distribution) Coordinator
 		BINDIR=$(BINROOT)/coord-oss
 		SRCDIR=coord
 		TARGET=$(BINDIR)/module-oss.so
-		PACKAGE_NAME=redisearch
+		PACKAGE_NAME=redisearch-community
```

Comment:
Need to verify this is the name for 2.10 branch

Reply:
On master, and 8.0 branches, we are using `redisearch-community` for versions compiled with `COORD=oss`